src/handlers/store-standardised-invoices/handler.ts

```diff
@@ -24,23 +26,52 @@ export const handler = async (event: SQSEvent): Promise<Response> => {
   if (rawInvoiceBucket === undefined || rawInvoiceBucket.length === 0)
     throw new Error("Raw invoice bucket not set.");
 
+  for (const records of event.Records) {
+    if (!records || !records.body) {
+      throw new Error("Record contains no body");
+    }
+  }
+
+  let recordBodies: StandardisedLineItem[] = [];
+  for (const records of event.Records) {
+    const recordList: StandardisedLineItem[] = JSON.parse(records.body);
+    recordBodies = recordBodies.concat(recordList);
+  }
```

Comment:
```suggestion
  const recordBodies = flatMap(record => JSON.parse(records.body))
```

might be a more javascripty way to do this